README.md

```diff
@@ -76,8 +76,8 @@ Populating the database
 
 For the initial run, you need to populate the CVE database by running:
 
+    ./sbin/db_mgmt_cpe_dictionary.py -p
     ./sbin/db_mgmt_json.py -p
-    ./sbin/db_mgmt_cpe_dictionary.py # This will take >45minutes on a decent machine, please be patient
```

Comment:
The CPE data needs to be filled first, because the CVE needs it for filling the information for vulnerable configurations.